pkg/core/native/native_test/neo_test.go

```diff
@@ -771,6 +771,36 @@ func TestNEO_CalculateBonus(t *testing.T) {
 	})
 }
 
+func TestNEO_UnclaimedGas(t *testing.T) {
+	// Initialize the test invoker for NEO validators.
+	neoValidatorsInvoker := newNeoValidatorsClient(t)
+	e := neoValidatorsInvoker.Executor
+
+	// Create a new account with no NEO balance.
+	acc := neoValidatorsInvoker.WithSigners(e.NewAccount(t))
+	accH := acc.Signers[0].ScriptHash()
+
+	t.Run("non-existing account", func(t *testing.T) {
+		// non-existing account, should return zero unclaimed GAS.
+		acc.Invoke(t, 0, "unclaimedGas", util.Uint160{}, 1)
+	})
+
+	t.Run("zero balance", func(t *testing.T) {
+		acc.Invoke(t, 0, "unclaimedGas", accH, 1)
+	})
```

Comment:
These two cases are the same from the contract's PoW, because no account record exists for these accounts. Merge them into a single "non-existing account" case.

Reply:
And "zero balance" case (L762, `if st.Balance.Sign() == 0 {`) is not tested at all, because to test it you need an existing account record with zero balance on it. Such record may be created by voting from account with zero NEO balance, vote for any registered candidate.